Makefile

```diff
@@ -59,7 +59,7 @@ IMAGE = $(REGISTRY)/$(IMGNAME)
 MULTI_ARCH_IMG = $(IMAGE)-$(ARCH)
 
 # Set default base image dynamically for each arch
-BASEIMAGE?=quay.io/kubernetes-ingress-controller/nginx-$(ARCH):0.53
+BASEIMAGE?=quay.io/kubernetes-ingress-controller/nginx-$(ARCH):0.54
```

Comment:
Please leave this as 0.53. First, we need to merge the change in the nginx image, publish and then open a PR to update the version (we don't merge PRs failing CI)

Reply:
Sure, done

Reply:
(And updated the description, as now this PR won't fix the issue, the next one will)